src/sentry/eventstream/kafka/protocol.py

```diff
@@ -28,13 +28,18 @@ def get_task_kwargs_for_insert(operation, event_data, task_state=None):
             "%Y-%m-%dT%H:%M:%S.%fZ",
         ).replace(tzinfo=pytz.utc)
 
+        def _get_attr(name):
+            if name == 'search_message':
+                name = 'message'
```

Comment:
I'm a little confused here. `Event.as_dict` has the following: 

```
data['message'] = self.real_message
```
Which seems to imply that the `message` attribute of the event dict/JSON body contains the `real_message` (which IIRC is the short non-concatenated version of the message for display?).  But here we are accessing that `message` property as if it was the `search_message`?

Reply:
It's still a bit confusing. `real_message` i only introduced temporarily to see where it breaks if i remove `message`. For all intends and purposes `Event.real_message` will go away later and become `Event.message` again.

About this particular code I'm not sure if it's correct. I will need to check that again.